app/controllers/admin/slides_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Admin::SlidesController < Admin::ApplicationController
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable @q [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)
* [InstanceVariableAssumption] assumes too much for instance variable @slide [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)

<!-- Comment by [SideCI](https://sideci.com) -->